include/vigra/config.hxx

```diff
@@ -149,7 +149,7 @@
 //                                                       //
 ///////////////////////////////////////////////////////////
 
```

Comment:
I don't know how to detect the platforms that need it without that test file.  The important factor is the user's particular copy of `libc++`, but not all versions of `libc++` are affected.  Do you know of a way to ask CMAKE what version of the C++ std library is active?